src/features/ajax/instrument/index.js

```diff
@@ -208,17 +208,27 @@ function subscribeToEvents (agentIdentifier, ee, handler, dt) {
   function onFetchBeforeStart (args) {
     var opts = args[1] || {}
     var url
-    // argument is USVString
     if (typeof args[0] === 'string') {
+      // argument is USVString
       url = args[0]
-      // argument is Request object
+
+      if (url.length === 0 && isBrowserScope) {
+        url = '' + globalScope.location.href
```

Comment:
No we should not short-circuit here. This is a valid request and we should handle it if distributed tracing is configured to do so.  The DT code later down the logic tree requires a value for the url to properly process. If fetch is called with an empty string, it will always use the current location as the URL.

Reply:
Good catch, didn't realize it would fallback to the window's location. In that same vein, there's one other path then that's not being honored in the current DT logic.  Fetch can accept any object that has a `toString()` method. --> https://developer.mozilla.org/en-US/docs/Web/API/fetch#parameters

So the following example wouldn't currently get decorated correctly by our DT process
`fetch({path: '/json', toString: () => '/json'})`

Reply:
True, let me add that before we ship this.

Reply:
Pushed.